api/geth_backend.go

```diff
@@ -113,7 +113,8 @@ func (b *GethStatusBackend) initialize() {
 	b.personalAPI = personalAPI
 	b.statusNode.SetMultiaccountsDB(b.multiaccountsDB)
 	b.log = log.New("package", "status-go/api.GethStatusBackend")
-	b.localPairing = false
+	b.localPairingManager = new(statecontrol.ProcessStateManager)
```

Comment:
Sounds like `localPairingStateManager` to me, wdyt?

Though having a `LocalPairingManager` I think is a good idea. I'will explain it below in comments.

Reply:
Yes I agree I'll make this change